### PR TITLE
Admin page (Table Status, Search by username)

### PR DESCRIPTION
**Pull request by 0tuedon**
PR Checklist

- [x]  Data Display in a Table: 
- [x]  Filtering Options 
- [x] Default View: 

**How to Test**
To test this pr make sure you are an admin and navigate to `reviews/all` or use the sidebar navigation.

### CODE REVIEW

File: src/pages/reviews/all.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+          >
+            {item}
+          </MenuItem>
+        ))}
+      </MenuList>
+    </Menu>
+  );
+};
+
+const Transactions = () => {
```
Comment (Extheoisah) [inline src/pages/reviews/all.tsx:73]: if this component is the same as the `Transactions` component, we should make it dynamic to avoid duplicate code

**Review by Extheoisah — CHANGES_REQUESTED** [review]

File: src/config/ui-config.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ export const ROUTES_CONFIG = {
   TRANSCRIPTS: "transcripts",
   WALLET: "wallet",
   TRANSACTIONS: "transactions",
+  ALL_REVIEWS: "reviews/all?status=active",
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/config/ui-config.ts:10]: One of the the requirement for issue #282  is to have a default view 
> | Default View: Upon accessing the page, the table should by default display 'Active' reviews.

File: src/components/tables/TableItems.tsx [hunk 3]
```diff
@@ -358,3 +359,52 @@ export const GroupedLinks = ({ data }: { data: ReviewTranscript }) => {
     </Flex>
   );
 };
+
+
+export const AdminGroupedLinks = ({ data }: { data: AdminReview }) => {
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/components/tables/TableItems.tsx:364]: Okay

**Comment by Extheoisah** [comment]
When I'm on the review page, the Review text doesn't have a background color to show that I'm on that route like we have for the Transactions page. We should use the same background color when the review page is in focus
image

**Comment by 0tuedon** [comment]
> When I'm on the review page, the Review text doesn't have a background color to show that I'm on that route like we have for the Transactions page. We should use the same background color when the review page is in focus image

This updated commit should fix for both transactions and reviews when there are filters

**Comment by 0tuedon** [comment]
> > When I'm on the review page, the Review text doesn't have a background color to show that I'm on that route like we have for the Transactions page. We should use the same background color when the review page is in focus ![image]


> This updated commit should fix for both transactions and reviews when there are filters

**Comment by 0tuedon** [comment]
> > > When I'm on the review page, the Review text doesn't have a background color to show that I'm on that route like we have for the Transactions page. We should use the same background color when the review page is in focus ![image]
> 
 This updated commit should fix for both transactions and reviews when there are filters

**Comment by 0tuedon** [comment]
> When I'm on the review page, the Review text doesn't have a background color to show that I'm on that route like we have for the Transactions page. We should use the same background color when the review page is in focus ![image]

> This updated commit should fix for both transactions and reviews when there are filters

**Comment by Extheoisah** [comment]
Can you add some padding in between "Transactions" and "Reviews"? No need to create a new commit. You can just amend the recent commit to include this change

Screenshot 2024-05-07 at 16 56 12

**Comment by 0tuedon** [comment]
> Reset Review: Leverage the existing /api/reviews/{id}/reset API endpoint to allow admins to reset an active review in the same way that admins can archive a transcript in the /transcripts page.
Additional Table Fields Based on Status
For Active reviews: Include the remaining time before the review expires.
For Pending reviews: Include a link to the pull request URL (pr_url) and the submission date.
For Merged reviews: Display the fields available in Pending reviews, plus the date the review was merged and a link to the final transcript.
Note: For dates displayed in the table, including the exact time is essential. To maintain a clean layout, we can show only the date initially, with the full date and time details appearing in a tooltip upon hovering over the date.

When merging pls note that there will be an additional PR for this additional issues.

**Review by Extheoisah — APPROVED** [review]

**Comment by Extheoisah** [comment]
merging now. @0tuedon Lets address the other issues in another pr

**Comment by kouloumos** [comment]
Great work @0tuedon! A couple of issues while reviewing this post-merged:
1. The status column is not useful as it is, "queued" or "not queued" is not useful information. The status should be displayed as one of the actual status with which you can filter (active, merged, etc.). This way when you search for a specific user/transcript, you can see the actual status of the previous claims.
2. Filtering should reset the url. As it is, it creates issues when for example you go on the second page of the table and then apply a filtering.
3. Updated At is not useful as a column. Let's remove it on the next update.
4. The Transactions table has a count of Results on the top right corner, let's also add this here
5. "Expired" filter also displays merged reviews.

**Pull request merged**